HoneyBadgers.RestApi/Services/ReportService.cs

```diff
@@ -61,5 +67,29 @@ public ReportGenreStats GetLastGeneratedReportGenreStats()
             return tuple;
         }
 
+        public bool StoreUserActivity(string actionArguments, string url, string userName, string userIpAdress, string HTTPMethod)
```

Comment:
Gdzie  to jest wywoływane? Czy zwracanie boola jest konieczne, może void będzie lepszy?

Reply:
poprawione